.github/ai-review/prompts/pr-summarizer.md

```
@@ -0,0 +1,101 @@
+## Walkthrough
+
+| File | Change | Summary |
+|------|--------|---------|
+<rows>
+
+## Sequence Diagram
+
+```mermaid
+<diagram>
+```
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)